src/plugins/intel_npu/README.md

```diff
@@ -166,6 +166,14 @@ The following properties are supported:
 | `ov::device::architecture`/</br>`DEVICE_ARCHITECTURE` | RO | Returns the platform information. | `N/A`| `N/A` |
 | `ov::device::full_name`/</br>`FULL_DEVICE_NAME` | RO | Returns the full name of the NPU device. | `N/A`| `N/A` |
 | `ov::internal::exclusive_async_requests`/</br>`EXCLUSIVE_ASYNC_REQUESTS` | RW | Allows to use exclusive task executor for asynchronous infer requests. | `YES`/ `NO`| `NO` |
+| `ov::device::type`/</br>`DEVICE_TYPE` | RO | Returns the type of device. See Type enum definition for possible return values | `N/A`| `N/A` |
```

Comment:
I believe we should write the supported values here instead of redirecting the user to a code snippet he may or may not know where it is. Otherwise we would have left code pointers to all properties in this table which would be less useful for casual users.

Reply:
added, thanks!